csharp/tests/ExecutionSingleizerTest.cs

```diff
@@ -138,8 +138,8 @@ public async Task ManyConcurrentExecutionShouldSucceed()
     for (var i = 0; i < n; ++i)
     {
       var j = results[i];
-      Assert.GreaterOrEqual(i,
-                            j);
+      Assert.That(j,
+                  Is.LessThanOrEqualTo(i));
```

Comment:
Less ?

Reply:
Yes, I inverted `i` and `j`. It makes more sense in the context of this test.